utils/loggingcpp/message.cpp

```diff
@@ -127,6 +127,30 @@ void Message::Args::add(uint64_t u64)
   fArgs.push_back(u64);
 }
 
+void Message::Args::add(int128_t i128)
```

Comment:
I sppose you can re-use [this routine.](https://github.com/mariadb-corporation/mariadb-columnstore-engine/blob/develop/datatypes/mcs_int128.cpp#L77)